options/locale/locale_zh-CN.ini

```diff
@@ -13,6 +13,7 @@ version=当前版本
 page=页面
 template=模板
 language=语言选项
+notifications = 通知
```

Comment:
hi,@appleboy @lunny this english key exist.
[https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini](https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini)

page = Page
template = Template
language = Language
notifications = Notifications
create_new = Create...